baselines/FedSMOO/FedSMOO/conf/fedavg.yaml

```diff
@@ -0,0 +1,47 @@
+---
+# ----------------------------------------------------------------------------
+## Hyperparams
+
+# lr_decay=0.998: SGD lr decay
+# beta: 10  # penalized coeff (select 10 for CIFAR-10 and 100 for CIFAR-100)
+# not used for some reason
+
+# ------------------------------------------------------------------------------
+
+num_rounds: 800    # number of rounds of communications (t in the paper)
+local_epochs: 5     # local inverval k in algorithm (CIFAR-10 -> 5, CIFAR-100 -> 2)
+batch_size: 50              # ( CIFAR-10 -> 50, CIFAR-100 -> 20)
+num_clients: 100           # 100 clients with 10% participation, 200 clients with 5% participation
+method: FedAvg
+
+dataset:
+  # dataset config (partitioning)
+  dataset_name: CIFAR10       # CIFAR 10/ CIFAR 100
+  dirichlet: false             # if not dirichlet then pathological
+  dirichlet_coeff: 0.1        # choose between 0.1, 0.6 
+  n_active_categories: 3      # 3,6 (cifar10), 20,10 (cifar100) # pathological
+
+model:
+  # model config
+  _target_: FedSMOO.models.ResNet18GN
+  num_classes: 10
+
+strategy:
+  #_target_: flwr.server.strategy.FedAvg
+  _target_: FedSMOO.strategy.FedAvgWithStragglerDrop
+  fraction_fit: 0.1       # 0.1, 0.05
+  fraction_evaluate: 0.0   # 0.1-0.2?
+  min_fit_clients: 2
+  min_evaluate_clients: 2
+  min_available_clients: 2
+
+client:
+  # client config
+  local_epochs: 5
+  learning_rate: 0.1  # local learning rate
+  lr_decay: 0.998     # 0.9995 in demo code
+  stragglers: 0.9
+
+client_resources:
+  num_cpus: 10
+  num_gpus: 1
```

Comment:
```suggestion
  num_cpus: 4
  num_gpus: 0.2
```

Reply:
When running your code i noticed clients just use 1GB of VRAM. I think it's fine if we allow for several clients to run in parallel on the same GPU. This can be done easily by lowering the fraction of GPU resources needed per client. I propose allowing to pack 5 (so `num_gpus=0.2`, since `1/0.2 = 5`). Also, let's lower CPU resources that each client needs. I don't think more than 4 cpus is really needed in this setting.

You can read more about how to set the `client_resources` effectively in the documentation: https://flower.dev/docs/framework/how-to-run-simulations.html#virtualclientengine-resources.